README.arm.md

```diff
@@ -1,14 +1,24 @@
-# Julia binaries for ARM
+# Julia on ARM (Linux)
 
-[Nightly builds](https://status.julialang.org/download/linux-arm) are
-available for ARMv7-A.
+Julia support for ARM is work-in-progress, though good results have been
```

Comment:
As far as I'm concerned there's one known issue https://github.com/JuliaLang/julia/issues/12346 on AArch32 and no known issue on AArch64 and all tests passes on AArch64 (for longer than a year now). Also, almost all x86 specific optimizations has also been implemented for AArch64 (with the exception of CPUID detection which is possible but there isn't any v8.1-v8.3 devices or devices with other interesting ISA extensions available for this to be used). I've also been routinely doing performance tests on AArch64 for a lot of my PRs.

While maybe AArch32 is still "work-in-progress" (especially armv6, I'd consider armv7 to be almost finished too) I'd consider AArch64 support to be finished at this point. This is also I didn't feel like adding AArch64 specific instructions.